assets/bingo/lang/pl_pl.json

```diff
@@ -85,198 +73,103 @@
 	"bingo.lobby.credits.community_and_support.issue_link": "Raporty błędów i sugestii (Angielski)",
 	"bingo.lobby.credits.community_and_support.discord_link": "Serwer Discord",
 	"bingo.lobby.credits.community_and_support.twitter_link": "Twitter (otwarte DMy, Angielski/Niemiecki)",
-
-	"bingo.lobby.credits.playtesters.sign.line1": "",
 	"bingo.lobby.credits.playtesters.sign.line2": "Ściana",
 	"bingo.lobby.credits.playtesters.sign.line3": "Testerów",
-	"bingo.lobby.credits.playtesters.sign.line4": "",
-
-	"bingo.lobby.credits.tools.sign.line1": "",
 	"bingo.lobby.credits.tools.sign.line2": "Użyte",
 	"bingo.lobby.credits.tools.sign.line3": "Narzędzia",
-	"bingo.lobby.credits.tools.sign.line4": "",
-
 	"bingo.lobby.credits.tools.vscode.sign.line3": "od",
 	"bingo.lobby.credits.tools.vscode.sign.line4": "Microsoft",
-
 	"bingo.lobby.credits.tools.datapack_helper_plus.sign.line3": "od",
 	"bingo.lobby.credits.tools.datapack_helper_plus.sign.line4": "SPGoding",
-
 	"bingo.lobby.credits.tools.minecraft_multi_noise_visualizer.sign.line3": "od",
 	"bingo.lobby.credits.tools.minecraft_multi_noise_visualizer.sign.line4": "Jochen Jacobs",
-
 	"bingo.lobby.credits.tools.nbt_explorer.sign.line3": "od",
 	"bingo.lobby.credits.tools.nbt_explorer.sign.line4": "Justin Aquadro",
-
 	"bingo.lobby.credits.tools.nbt_studio.sign.line3": "od tryashtar",
-	"bingo.lobby.credits.tools.nbt_studio.sign.line4": "",
-
 	"bingo.lobby.credits.tools.nbt_viewer.sign.line3": "od Misode",
-	"bingo.lobby.credits.tools.nbt_viewer.sign.line4": "",
-
-
-	"bingo.lobby.card_generation.random_card.sign.line1": "",
 	"bingo.lobby.card_generation.random_card.sign.line2": "Wygeneruj",
 	"bingo.lobby.card_generation.random_card.sign.line3": "Losową Kartę",
-	"bingo.lobby.card_generation.random_card.sign.line4": "",
-
-	"bingo.lobby.card_generation.from_seed.sign.line1": "",
 	"bingo.lobby.card_generation.from_seed.sign.line2": "Wygeneruj Kartę",
 	"bingo.lobby.card_generation.from_seed.sign.line3": "Z Ziarna",
-	"bingo.lobby.card_generation.from_seed.sign.line4": "",
-
-	"bingo.lobby.card_generation.start_game.sign.line1": "",
 	"bingo.lobby.card_generation.start_game.sign.line2": "Start",
 	"bingo.lobby.card_generation.start_game.sign.line3": "Gry",
-	"bingo.lobby.card_generation.start_game.sign.line4": "",
-
 	"bingo.lobby.card_generation.crafting_table.sign.line1": "Wyszukiwarka",
 	"bingo.lobby.card_generation.crafting_table.sign.line2": "Receptur",
 	"bingo.lobby.card_generation.crafting_table.sign.line3": "Rzemieślni-",
 	"bingo.lobby.card_generation.crafting_table.sign.line4": "czych",
-
 	"bingo.lobby.card_generation.furnace.sign.line1": "Wyszukiwarka",
 	"bingo.lobby.card_generation.furnace.sign.line2": "Receptur",
 	"bingo.lobby.card_generation.furnace.sign.line3": "Przetapiania",
-	"bingo.lobby.card_generation.furnace.sign.line4": "",
-
+	"bingo.lobby.card_generation.furnace.sign.line4": " ",
```

Comment:
Here is an example where I needed to manually translate as space to make it work ;)